web/src/app/shared/pipes/date-time-zone/date-time-zone.pipe.ts

```diff
@@ -8,11 +8,15 @@ export class DateTimeZonePipe implements PipeTransform {
 
   private datesMap = new Map<string, string>();
 
-  transform(value: any, args?: any): any {
+  transform(value: any, displayTime: boolean = true, args?: any): any {
 
     let date = this.datesMap.get(value);
     if (!date) {
-      date = TaskanaDate.getDateToDisplay(value);
+      if (displayTime) {
+        date = TaskanaDate.getDateToDisplay(value);
+      } else {
+        date = TaskanaDate.convertSimpleDate(new Date (value));
```

Comment:
Convert simple date doesn't apply timeZone, if you have a look to getDateToDisplay it applies time zone explicitly.
you have to change getDateToDisplay method adding a new argument, dateFormat for instance and apply the format desired to applyTimeZone,

this is a suggestion:

`
public static dateFormatWithTime: string = 'yyyy-MM-dd HH:mm:ss'
public static dateFormat: string = 'yyyy-MM-dd'

public static getDateToDisplay(date: string, dateFormat: string = this.dateFormatWithTime): string {
    return this.applyTimeZone(date, dateFormat);
  }

  public static applyTimeZone(date: string): string | null {
    const dateFormat = 'yyyy-MM-dd HH:mm:ss';
    const dateLocale = 'en-US';
    const datePipe = new DatePipe(dateLocale);
    return datePipe.transform(date, dateFormat, Intl.DateTimeFormat().resolvedOptions().timeZone);
  }
`